index.js

```diff
@@ -95,14 +95,16 @@ module.exports.process = (output, origin) => {
   const subtitleFile = findBestSubtitleFile(subtitles) || findBestSubtitleFile(automatic_captions);
   const subtitleUrl = subtitleFile ? `${origin}/ytdl/vtt?suburi=${encodeURIComponent(subtitleFile.subs.url)}` : '';
   const title = data.title || '';
+  const thumbnail = data.thumbnail || '';
```

Comment:
Is this just stored as a buffer or is this a URL to the actual thumbnail file?

Reply:
It's just the url, we actually need to proxy it through our backend for client to have access to it.

Reply:
Yeah cool, was just making sure we aren't loading full image buffers in

Reply:
Even if we were, to optionally load it we have to change vivi-service-ytdl cuz the request is processed there.